templates/inspec/singular_resource.erb

```diff
@@ -101,4 +101,8 @@ class <%= object.name -%> < Inspec.resource(1)
   def exists?
     !@fetched.nil?
   end
+<% unless object&.additional_functions.nil? -%>
+
+<%= lines(indent(object.additional_functions, 2)) -%>
```

Comment:
Idea (because I don't think we have anything like this): what if we had a function that, given a list, would output things with the proper number of spaces (no spaces for last one, extra x lines in between all others)? It would fix this particular problem and could be used elsewhere

Reply:
Would we pass the entire class to that? The newline here is so that there will be a space between the end of the exists? function and the start of whatever custom code we add.